test/directives/unroll.f90

```diff
@@ -74,20 +72,19 @@ subroutine func3(m, a, b)
   ! CHECK:              br i1 {{.*}}, label %[[BB3]]
   ! CHECK-O0-SAME:      !llvm.loop [[MD_LOOP3:![0-9]+]]
   ! CHECK-O1-SAME:      !llvm.loop [[MD_LOOP3:![0-9]+]]
-  ! CHECK-DISABLED-NOT: !llvm.loop !{{[0-9]+}}
 end subroutine func3
 
-! CHECK-O0: [[MD_LOOP1]] = distinct !{[[MD_LOOP1]], [[MD_ENABLE:![0-9]+]]}
-! CHECK-O0: [[MD_ENABLE]] = !{!"llvm.loop.unroll.enable"}
+! CHECK-O0: [[MD_ENABLE:![0-9]+]] = !{!"llvm.loop.unroll.enable"}
+! CHECK-O0: [[MD_LOOP1]] = distinct !{[[MD_LOOP1]], {{.*}}, {{.*}}, [[MD_ENABLE]]}
```

Comment:
Can these lines use the FileCheck directive `CHECK-O0-DAG`? In my downstream build, the test outputs these lines in a different order:
```
!15 = !{!15, !12, !14, !16}
!16 = !{!"llvm.loop.unroll.enable"}
```

Reply:
Done, I hope I got it right? :)

Reply:
Yes, thanks.